linode/provider.go

```diff
@@ -170,6 +170,7 @@ func Provider() *schema.Provider {
 			"linode_instance_backups":       backup.DataSource(),
 			"linode_instance_type":          instancetype.DataSource(),
 			"linode_instance_types":         instancetypes.DataSource(),
+			"linode_instance_ip":            instanceip.DataSource(),
```

Comment:
We might want to make this into `linode_instance_networking` since `linode_instance_ip` refers to a single instance of a Linode IP. Definitely open to discussion though 🙂